wolfssl/quic.h

```diff
@@ -288,6 +288,10 @@ int wolfSSL_quic_hkdf(uint8_t* dest, size_t destlen,
                       const uint8_t* salt, size_t saltlen,
                       const uint8_t* info, size_t infolen);
 
+#ifdef WOLFSSL_CALLBACKS
```

Comment:
I would prefer this in the settings.h wrapped with `#ifdef WOLFSSL_QUIC`. Or at the top of this file, not the bottom

Reply:
done.  moved to `settings.h` -- there was precedent for this sort of check there.